vamp/artifacts/breeds/quantification.yaml

```diff
@@ -8,19 +8,19 @@ environment_variables:
   VAMP_URL                            : ${config://vamp.workflow-driver.workflow.vamp-url}
   VAMP_API_TOKEN                      : ${vamp://token}
   VAMP_NAMESPACE                      : ${config://vamp.namespace}
-  VAMP_WORKFLOW_EXECUTION_TIMEOUT     : 10
+  VAMP_WORKFLOW_EXECUTION_TIMEOUT     : 60
   VAMP_KEY_VALUE_STORE_CONNECTION     : ${config://vamp.workflow-driver.workflow.vamp-key-value-store-connection}
   VAMP_KEY_VALUE_STORE_TOKEN          : ${config://vamp.workflow-driver.workflow.vamp-key-value-store-token}
   VAMP_KEY_VALUE_STORE_PATH           : ${config://vamp.workflow-driver.workflow.vamp-key-value-store-path}
-  VAMP_WORKFLOW_EXECUTION_PERIOD      : 10
+  VAMP_WORKFLOW_EXECUTION_PERIOD      : 60
   VAMP_KEY_VALUE_STORE_TYPE           : ${config://vamp.workflow-driver.workflow.vamp-key-value-store-type}
   VAMP_PULSE_ELASTICSEARCH_URL        : ${config://vamp.pulse.elasticsearch.url}
   VAMP_HEALTH                         : true
   VAMP_ELASTICSEARCH_HEALTH_INDEX     : ${es://health}
-  VAMP_HEALTH_TIME_WINDOW             : 30
+  VAMP_HEALTH_TIME_WINDOW             : 60
   VAMP_METRICS                        : true
   VAMP_ELASTICSEARCH_METRICS_INDEX    : ${es://metrics}
-  VAMP_METRICS_TIME_WINDOW            : 30
+  VAMP_METRICS_TIME_WINDOW            : 60
```

Comment:
When we create quant workflow from Lifter, the default value of metrics and health windows is 500: https://github.com/magneticio/assembly/blob/9b6c8ee8f39e03218ec08bed14eb9b000b14fd62/images/base-vamp-ee-lifter/artifacts/breeds/quantification.yml 
Maybe we should set the same value here?